frontend/src/components/accounts/AccountDetails.tsx

```diff
@@ -23,285 +25,304 @@ class AccountDetails extends React.Component<Props> {
   render() {
     const { account, currentNearNetwork } = this.props;
     return (
-      <div className="account-info-container">
-        <Row noGutters>
-          <Col
-            xs="12"
-            md={typeof account.storageUsage === "undefined" ? "12" : "4"}
-          >
-            <CardCell
-              title={
-                <Term
-                  title={"Transactions"}
-                  text={"Total transaction sent and received by this account. "}
-                  href={"https://docs.near.org/docs/concepts/transaction"}
+      <Translate>
+        {({ translate }) => (
+          <div className="account-info-container">
+            <Row noGutters>
+              <Col
+                xs="12"
+                md={typeof account.storageUsage === "undefined" ? "12" : "4"}
+              >
+                <CardCell
+                  title={
+                    <Term
+                      title={translate("model.transactions.title").toString()}
+                      text={translate(
+                        "component.accounts.AccountDetails.transactions.text"
+                      ).toString()}
```

Comment:
Is it required to use `.toString()` explicitly? In [Search component you skipped it](https://github.com/near/near-explorer/pull/679/files?diff=split&w=1#diff-7af4e8dfed46b64428f48a72bc152ce7ea0f04b2992ec9c797ab6106765faa3dR87), and it seems to be working fine

Reply:
It's not required to use `toString()` for `text` property of `<Term />`, but it's required to convert to string for `title` as defined in <Term /> component. 

```ts
interface Props {
  title: string;
  text: string | React.ReactNode;
  href?: string;
}
```

We can update <Term /> componnet or create another <TermI18N /> component to reduce those redundant i18n related processing.

Reply:
I see. I am fine with this explicitness, just wanted to double-check. I am fine to lift the type requirements from just `string` to whatever is needed. I don't think we need `TermI18N` component, though